### DISCUSSION TITLE
Multiple WSL Configurations

### DISCUSSION THREAD

**Discussion by mshawnreynolds** [Eugeny/tabby#3167] [Q&A]
This there a way to specify which WSL shell I can use in the profile. I have Kali and Ubuntu installed and open them separately in Windows Terminal using a GUID. Can I do this with Terminus?

**Comment by Eugeny** [comment]
Yes - you should see separate WSL distros as separate entries both in Settings -> Shell and in the profile dropdowns.

**Reply by mshawnreynolds** [reply]
I am only seeing one.

**Comment by jpwilksch** [comment]
Each WSL2 distro has an executable under `C:\Users\<username>\AppData\Local\Microsoft\WindowsApps\`. So, for example, my Ubuntu shell can be launched with `C:\Users\<username>\AppData\Local\Microsoft\WindowsApps\ubuntu.exe`.

Create a profile and specify this as the `command`.

Alternatively, provide the distro name as a parameter to the `wsl` executable:
`C:\Windows\System32\wsl.exe -d ubuntu`

**Comment by tsurgej** [comment]
I am not really able to configure it with the wsl options. 
It only works with profile "Custom shell" and the path "c:\windows\system32\wsl.exe -d Ubuntu-20.04". I wonder how I can set the starting directory.
All other wsl options provided do not open a shell, the tabs immediatly close. Even the builtin default distro app option. I copied the setting from other terminal apps where I got it running.
